src/js/structs/JobRun.js

```diff
@@ -2,7 +2,7 @@ import DateUtil from '../utils/DateUtil';
 import Item from './Item';
 import JobTaskList from './JobTaskList';
 
-module.exports = class JobActiveRun extends Item {
+class JobRun extends Item {
   getDateCreated() {
```

Comment:
Thats, true! But luckily they all use `createdAt` and that's the only date we're using at the moment. I'll add the rest (e.g. `status`) in the struct or `MetronomeUtil`, which should enable us to use the very same struct for all runs.